api/src/test/java/org/mandulis/mts/controller/AttachmentControllerIT.java

```diff
@@ -0,0 +1,136 @@
+package org.mandulis.mts.controller;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mandulis.mts.entity.Attachment;
+import org.mandulis.mts.exception.AttachmentWithoutNameException;
+import org.mandulis.mts.exception.EmptyAttachmentException;
+import org.mandulis.mts.repository.AttachmentRepository;
+import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Random;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.multipart;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class AttachmentControllerIT {
+
+    @Value("${storage.location}")
+    private String storageLocation;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
```

Comment:
Field injection is not recommended

Reply:
What's the difference for tests? Plus @MockBeans are kinda hard to inject through constructors :D